util/parsenum.h

```diff
@@ -107,26 +108,38 @@ _Pragma("clang diagnostic pop")
  * For an unsigned integer variable ${x}, this can also be invoked as
  * PARSENUM_BASE(x, s, base), in which case the minimum and maximum values are
  * set to the limits of the unsigned integer type.
+ *
+ * For a floating-point variable ${x}, the ${base} must be 0.
```

Comment:
I don't know if this comment should be worked into the previous paragraph, or if it can stay here by itself.

Reply:
This is fine.